### PR TITLE
Prepare v2 release

### PR DESCRIPTION
**Pull request by darosior**
This is assuming #630 is merged, but not the fix for #323 (can be added to the CHANGELOG if it does).

### CODE REVIEW

**Comment by edouardparis** [comment]
seems good to me, can you `cargo update -p liana` for the gui in an other commit ?

**Comment by darosior** [comment]
I'll do it in another PR once it's merged? Also probably best to wait for branch-off?
------- Original Message -------

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+
+- The UX for creating a descriptor where a signer is present in different spending paths was
+  significantly improved.
+- The installation process, as well as the usage of the wallet, was made more user friendly by
+  optionally encapsulating the management (download, start and stop) of the `bitcoind`.
+
+### Fixes
+
+- We more gracefully stop the Bitcoin backend poller when the block chain is still in the process of
+  being synchronized. This would previously appear to hand and could freeze the GUI.
```
Comment (jp1ac4) [inline CHANGELOG.md:31]: nit: "hand" -> "hang" ?

Comment (darosior) [reply CHANGELOG.md]: Thanks, done.

**Comment by darosior** [comment]
self-ACK edb2c513a3fbd6a002ba94425430be8ab34258fd -- simply docs and a version bump

**Pull request merged**